workflow/applications/gefs.py

```diff
@@ -12,28 +13,35 @@ def __init__(self, conf: Configuration):
 
         base = conf.parse_config('config.base')
         self.run = base.get('RUN', 'gefs')
+        self.runs = [self.run]
 
-    def _get_app_configs(self):
+    def _netmode_run_options(self, base: Dict[str, Any], run_options: Dict[str, Any]) -> Dict[str, Any]:
+
+        run_options[self.run]['nens'] = base.get('NMEM_ENS', 0)
```

Comment:
Instead of having a separate method, why not just overload the original?

```suggestion
    def _get_run_options(self, conf: Configuration) -> Dict[str, Any]:
        run_options = super()._get_run_options(conf)
        for run in self.runs:
            run_options[self.run]['nens'] = base.get('NMEM_ENS', 0)
```

Reply:
I think that will work.  Thanks!